modules/cassandra/src/main/java/org/apache/ignite/cache/store/cassandra/CassandraCacheStore.java

```diff
@@ -105,7 +111,7 @@ public CassandraCacheStore(DataSource dataSrc, KeyValuePersistenceSettings setti
             for (Future<?> fut : futs)
                 U.get(fut);
 
-            if (log != null && log.isDebugEnabled() && storeSes != null)
+            if (log.isDebugEnabled() && storeSes != null)
```

Comment:
It will produce NullPointerException if Ignite was started with Basic configuration without Spring support added - in a such case IgniteLogger simply will not be injected. That's why I added the check " log != null "